sotodlib/core/axisman_io.py

```diff
@@ -13,6 +15,11 @@
 from .axisman import *
 from .flagman import FlagManager
 
+# Backwards compatibility for before skip_shape_check was added
+RangesMatrix = so3g.proj.RangesMatrix
+if "skip_shape_check" in inspect.signature(so3g.proj.RangesMatrix).parameters:
+    RangesMatrix = partial(so3g.proj.RangesMatrix, skip_shape_check=True)
+
```

Comment:
Instead of replacing RangesMatrix class with whatever partial(...) would give you, please just create var
```
_rm_fast_kwargs = {}
if "skip_shape_check" ...:
  _rm_fast_kwargs = {'skip_shape_check': True}
```
Then pass those to usual class.